CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
-# dbt_iterable v0.RELEASE.RELEASE
+# dbt_iterable v0.12.0
+[PR #44](https://github.com/fivetran/dbt_iterable/pull/44) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+- Introduces variable `iterable__using_event_extension` to allow the `event_extension` table to be disabled and exclude its field, `experiment_id`, from persisting downstream. This permits the downstream models to run even if the source `event_extension` table does not exist. By default the variable is set to True. If you don't have this table, you will need to set `iterable__using_event_extension` to False. For more information on how to configure the `iterable__using_event_extension` variable, refer to the [README](https://github.com/fivetran/dbt_iterable/blob/main/README.md#step-4-enablingdisabling-models). 
```

Comment:
You'll also want to add in here that the uniqueness test has been updated to account for instances when`iterable__using_event_extension` is enabled/disabled. It would be good to add what the new surrogate fields are and which models they can be found in.

Reply:
updated!